### DISCUSSION TITLE
Ability to post-process generation data

### DISCUSSION THREAD

**Discussion by denisergashbaev** [langfuse/langfuse#1097] [Ideas]
### Describe the feature or potential improvement

It would be nice to be able to postprocess generation info to remove some of the information while keeping the other data (cost, time, etc)

### Contribute

- [ ] Yes, I can implement this and raise a PR

**Comment by marcklingen** [comment]
Thanks for sharing, can you share more ideas on how this could look like when using the sdks/integrations? How would you like to filter/drop data from traces/observations?
